core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/ui/dot-experiments-goal-configuration-url-parameter-component/dot-experiments-goal-configuration-url-parameter-component.component.scss

```diff
@@ -0,0 +1,38 @@
+@use "variables" as *;
+
+.wrapper {
+    gap: $spacing-4;
+}
+
+.field {
+    margin: $spacing-0;
+
+    gap: $spacing-1;
+    margin: $spacing-0;
+
+    label {
+        font-weight: $font-weight-bold;
+        margin-bottom: $spacing-0;
+        font-size: $font-size-md;
+    }
+
+    p {
+        margin: $spacing-0;
+        color: $color-palette-gray-500;
+    }
+
+    h3 {
+        font-size: $font-size-md;
+        margin: $spacing-0;
+        margin-bottom: $spacing-1;
+        font-weight: $font-weight-bold;
+    }
+}
```

Comment:
We already have a `.field` class that style the labels... we need to keep the forms labels consistent.

Reply:
Let me take a look.